File: codingame/onboarding.cs
using System;
class Player{
	static void Main(string[] args){
		for(;;){
			string enemy1 = Console.ReadLine(); // name of enemy 1
			int dist1 = int.Parse(Console.ReadLine()); // distance to enemy 1
			string enemy2 = Console.ReadLine(); // name of enemy 2
			int dist2 = int.Parse(Console.ReadLine()); // distance to enemy 2

			Console.WriteLine(dist1<dist2 ? enemy1 : enemy2);
		}
	}
}

File: codingame/onboarding.vb
module Player
	sub Main()
		while True
			Dim enemy1 as String
			enemy1 = Console.ReadLine() ' name of enemy 1
			Dim dist1 as Integer
			dist1 = Console.ReadLine() ' distance to enemy 1
			Dim enemy2 as String
			enemy2 = Console.ReadLine() ' name of enemy 2
			Dim dist2 as Integer
			dist2 = Console.ReadLine() ' distance to enemy 2
			Console.WriteLine(iif(dist1<dist2,enemy1,enemy2))
		end while
	end sub
end module